File: configlibrary/src/main/java/com/shmedo/configlibrary/iot/model/das/MemsBean.java
package com.shmedo.configlibrary.iot.model.das;

/**
 * 创建者:   gonghe <br/>
 * 创建时间:  2021/4/16 <br/>
 * 描述：  DAS 状态页面MEMS传感器状态
 */
public class MemsBean {
    private int type;
    private String vaule;
    private int errno;

    public int getType() {
        return type;
    }

    public void setType(int type) {
        this.type = type;
    }

    public String getVaule() {
        return vaule;
    }

    public void setVaule(String vaule) {
        this.vaule = vaule;
    }

    public int getErrno() {
        return errno;
    }

    public void setErrno(int errno) {
        this.errno = errno;
    }
}

File: lib_device_base/src/main/java/com/shmedo/lib/device/base/iot_cmd/model/das/MemsBean.kt
package com.shmedo.lib.device.base.iot_cmd.model.das

import com.squareup.moshi.JsonClass

/**
 * 创建者:   gonghe <br></br>
 * 创建时间:  2021/4/16 <br></br>
 * 描述：  DAS 状态页面MEMS传感器状态
 */
@JsonClass(generateAdapter = true)
data class MemsBean(
    var type: String = "",
    var vaule: String = "",
    var errno: String = "",
)